portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -31,6 +31,7 @@
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         UserService userService = UserServiceFactory.getUserService();
         String urlToRedirectTo = "/index.html";
+        response.setContentType("application/json;"); 
```

Comment:
I guess we misses this in the previous PR. :-( Please explain this change in this PR's description.

Reply:
The PR description is updated! :)